internal/output/progress_win.go

```diff
@@ -47,7 +47,7 @@ func (d *Spinner) moveCaretBackInCommandPrompt(n int) {
 		cursor.x = csbi.cursorPosition.x + short(-n)
 		cursor.y = csbi.cursorPosition.y
 
-		r2, _, err2 := procSetConsoleCursorPosition.Call(uintptr(handle), uintptr(*(*int32)(unsafe.Pointer(&cursor))))
+		r2, _, err2 := procSetConsoleCursorPosition.Call(uintptr(handle), uintptr(*((*uint32)(unsafe.Pointer(&cursor)))))
```

Comment:
I'm not sure how this got "autoformatted", but it did, and it does look worth keeping. This is the only place in the code that this is done, so it's not out of sync with anything else.

Reply:
Please fix your PR so this isn't in the history.